.github/workflows/release.yml

```diff
@@ -63,6 +63,8 @@ jobs:
       - name: Install GTK4 (Windows)
         run: |
           pipx install gvsbuild
+          # Use stable Rust for gvsbuild to avoid issues with project's nightly toolchain
+          $env:RUSTUP_TOOLCHAIN = "stable"
```

Comment:
Wierd. Dont have a windows machine, can you let me know what are issues you saw here ?